packages/scrollbar/src/index.vue

```diff
@@ -28,7 +28,7 @@
 <script lang="ts">
 import { addResizeListener, removeResizeListener } from '@element-plus/utils/resize-event'
 import { addUnit, isArray, isString, toObject } from '@element-plus/utils/util'
-import { computed, defineComponent, nextTick, onBeforeUnmount, onMounted, provide, ref } from 'vue'
+import { computed, CSSProperties, defineComponent, nextTick, onBeforeUnmount, onMounted, PropType, provide, ref } from 'vue'
```

Comment:
You need to use this line for type importing, this is the convention of our repo.

```ts
import type { CSSProperties, PropType } from 'vue'
```

Reply:
Add this after the last line of importing actual object.